### DISCUSSION TITLE
ST7262 Support

### DISCUSSION THREAD

**Discussion by nosowc** [moononournation/Arduino_GFX#390] [General]
Hello there!

I have a display with an ST7262 IC that I wanted to code through Arduino. I was looking through this library and, while it mentions it supports this IC driver in the Arduino library description, I see no mention of it within the GitHub documentation outside of the demo video on YouTube. I was just wondering if this library still supports that driver? Thanks!

**Comment by moononournation** [comment]
it is supported RGB interface, you can use RGB Panel class to use it.

**Reply by aremmell** [reply]
I am attempting to use `Arduino_RGB_Display` right now to initialize a Newhaven 800x480 display with the ST7262 using the following code with no luck (display does not turn on):

```cpp
auto expander = new Arduino_XCA9554SWSPI(
    PCA_TFT_RESET, PCA_TFT_CS, PCA_TFT_SCK, PCA_TFT_MOSI, &Wire, 0x3C
);
auto rgbpanel = new Arduino_ESP32RGBPanel(
    TFT_DE, TFT_VSYNC, TFT_HSYNC, TFT_PCLK, TFT_R1, TFT_R2, TFT_R3, TFT_R4, TFT_R5,
    TFT_G0, TFT_G1, TFT_G2, TFT_G3, TFT_G4, TFT_G5, TFT_B1, TFT_B2, TFT_B3, TFT_B4, TFT_B5,
    0 /* hsync_polarity */, 8 /* hsync_front_porch */, 4 /* hsync_pulse_width */, 8 /* hsync_back_porch */,
    0 /* vsync_polarity */, 8 /* vsync_front_porch */, 4 /* vsync_pulse_width */, 8 /* vsync_back_porch */);
auto display = std::make_shared<Arduino_RGB_Display>(
    DISPLAY_WIDTH, 
    DISPLAY_HEIGHT, 
    rgbpanel, 
    0, 
    false, 
    expander, 
    GFX_NOT_DEFINED, 
    nullptr, 
    0
);
```

**Reply by moononournation** [reply]
Your display have XCA9554 chip?

**Reply by aremmell** [reply]
I got it to turn on using the Adafruit RA8875. I guess it just doesn't work with the Qualia RGB 666.
